services/web/server/src/simcore_service_webserver/catalog/_constants.py

```diff
@@ -3,3 +3,5 @@
 MSG_CATALOG_SERVICE_UNAVAILABLE: Final[
     str
 ] = "Currently catalog service is unavailable, please try again later"
+
+MSG_CATALOG_SERVICE_NOT_FOUND: Final[str] = "Not Found"
```

Comment:
Maybe a add bit more information here, to offer the caller some more insight?

Reply:
Agree with @GitHK  :-)